src/test/kotlin/racingcar/domain/CarTest.kt

```diff
@@ -26,14 +31,21 @@ internal class CarTest {
         assertThrows<IllegalArgumentException> { Car("가나다라마바") }
     }
 
-    @Test
-    internal fun `자동차는 포지션이 있어야한다`() {
+    @ParameterizedTest
+    @MethodSource("provideCarPosition")
+    internal fun `자동차는 포지션이 있어야한다`(shouldMove: Boolean, expectedPosition: Int) {
         val car = Car("test")
-        car.move()
-        car.stop()
-        car.move()
-        assertThat(car.currentPosition(1)).isEqualTo(1)
-        assertThat(car.currentPosition(2)).isEqualTo(1)
-        assertThat(car.currentPosition(3)).isEqualTo(2)
+        car.moveOrStop(shouldMove = shouldMove)
+        assertEquals(expectedPosition, car.position(attempt = 1))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        assertThat(car.position(1)).isEqualTo(expectedPosition)
```